### PR TITLE
Code Readability - Doors

### PR DESCRIPTION
**Pull request by KalevTait**
## What Does This PR Do
Converts 0/1 to FALSE/TRUE where those #defines make the code more readable.

## Why It's Good For The Game
More readable code is easier to work with.

### CODE REVIEW

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -211,7 +211,7 @@
 
 	for(var/obj/machinery/treadmill_monitor/T in targets)
 		T.total_joules = 0
-		T.on = 1
+		T.on = TRUE
 
 	return 1
```
Comment (Sirryan2002) [inline code/game/machinery/doors/brigdoors.dm:216]:
```suggestion
	return TRUE
```
I'm assuming these are supposed to be bools as well. All instances of this proc being used don't actually check the value returned so there's not really much point, but code readability.

File: code/game/machinery/doors/brigdoors.dm [hunk 2]
```diff
@@ -244,13 +244,13 @@
...
+		C.locked = FALSE
 		C.icon_state = C.icon_closed
 
 	for(var/obj/machinery/treadmill_monitor/T in targets)
 		if(!T.stat)
 			T.redeem()
-		T.on = 0
+		T.on = FALSE
 
 	return 1
```
Comment (Sirryan2002) [inline code/game/machinery/doors/brigdoors.dm:255]:
```suggestion
	return TRUE
```

File: code/game/machinery/doors/door.dm [hunk 3]
```diff
@@ -256,7 +256,7 @@
 		flick("door_spark", src)
 		sleep(6)
 		open()
-		emagged = 1
+		emagged = TRUE
 		return 1
```
Comment (Sirryan2002) [inline code/game/machinery/doors/door.dm:260]:
```suggestion
		return TRUE
```

File: code/game/machinery/doors/brigdoors.dm (hunk 1 above)
Comment (KalevTait) [reply code/game/machinery/doors/brigdoors.dm:216]: Yeah, I wasn't going to go there, as that's another big can of worms that I know once I started in one spot I'd want to keep doing everywhere.... but just accepting your suggestions, I can probably do that with out feeling like I need to do a find in files for 'return 1'. 😛

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**